src/lib/models/sml-versions.ts

```diff
@@ -1,25 +1,41 @@
 import * as zod from 'zod';
 import type { VersionStabilities } from '$lib/generated';
 
+export type SMLLinkData = {
+  id: string;
+  SMLVersionLinkID: string;
+  platform: string;
+  link: string;
+};
+
 export type SMLVersionData = {
   version: string;
   satisfactory_version: number;
   bootstrap_version?: string;
   stability: VersionStabilities;
   link: string;
+  arch: SMLLinkData[];
   changelog: string;
   date: string;
 };
 
 const versionRegex =
   /^(<=|<|>|>=|\^)?(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/;
 
+export const smlLinksSchema = zod.object({
```

Comment:
Please rename to `smlArchsSchema`

Reply:
https://github.com/satisfactorymodding/smr-frontend/pull/47/commits/9c0893e0353c97b347f9f0152ec590c614d255ac